src/templates/base.html

```diff
@@ -31,9 +31,26 @@
     <div id="navigation">
       {% block navigation %}{% endblock %}
     </div>
+    <select class="language-switcher">
+      <option
+        class="default-language"
+        selected="selected"
+        value={{ self.lang()[0:2] }}>
+      </option>
+      {% for language in supported_languages %}
+        {% if language != self.lang()[0:2] %}
```

Comment:
I think we only support 2 character language codes, so the `[0:2]` slice shouldn't be necessary.
```suggestion
        {% if language != self.lang() %}
```

Reply:
@rviscomi By using `self.lang()` we are reading the value that is passed into this [block](https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/src/templates/base.html#L2). 

Both `base.html` files of [en](https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/src/templates/en/base.html#L3) and [ja](https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/src/templates/ja/base.html#L3) are not in 2 character language codes, thus I included the slice.

Reply:
Oh I see, thanks for clarifying.